<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionExchangeWidget</class>
 <widget class="QWidget" name="FunctionExchangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>130</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_normal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: bold 16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>兑换合约</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="onchainOrderBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>130</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OTC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exchangeBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>130</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>交易所</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contractTokenBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>130</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标准代币合约</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_normal_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: bold 16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>DAPP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="nftTokenBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>130</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NFT</string>
   </property>
  </widget>
  <zorder>exchangeBtn</zorder>
  <zorder>label_normal</zorder>
  <zorder>onchainOrderBtn</zorder>
  <zorder>contractTokenBtn</zorder>
  <zorder>label_normal_2</zorder>
  <zorder>nftTokenBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
